spec/fixtures/iex/client/search/missing.yml

```diff
@@ -0,0 +1,49 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://sandbox.iexapis.com/v1/search/?token=Tsk_341c973d296e4e18aa61dd63050ce235
```

Comment:
Remove your actual token from here.

Reply:
SO DUMB! thanks. I regenerated my tokens just to be safe